packages/edit-site/src/components/global-styles/screen-css.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { __experimentalVStack as VStack } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import ScreenHeader from './header';
+import Subtitle from './subtitle';
+import CustomCSSControl from './custom-css';
+
+function ScreenCSS() {
+	return (
+		<>
+			<ScreenHeader
+				title={ __( 'CSS' ) }
+				description={ __(
+					'Add CSS beyond the styles offered by the existing design tools.'
```

Comment:
```suggestion
					'Add your own CSS to customize the appearance and layout of your site.'
```

Reason: to remove jargon and ambiguous preposition (beyond)

Reply:
done.